Code/Source/RobotImporter/URDF/URDFPrefabMaker.cpp

```diff
@@ -133,13 +139,17 @@ namespace ROS2
 
     void URDFPrefabMaker::AddRobotControl(AZ::EntityId rootEntityId)
     {
-        // TODO - check for RigidBody
-        ControlConfiguration controlConfiguration;
-        controlConfiguration.m_robotConfiguration.m_body = rootEntityId;
-        controlConfiguration.m_broadcastBusMode = false;
-        controlConfiguration.m_topic = "cmd_vel";
-        AZ::Entity* rootEntity = AzToolsFramework::GetEntityById(rootEntityId);
-        rootEntity->CreateComponent<ROS2RobotControlComponent>(controlConfiguration);
+        const auto componentId = Utils::CreateComponent(rootEntityId, ROS2RobotControlComponent::TYPEINFO_Uuid());
+        if (componentId)
+        {
+            ControlConfiguration controlConfiguration;
+            controlConfiguration.m_robotConfiguration.m_body = rootEntityId;
+            controlConfiguration.m_broadcastBusMode = false;
+            controlConfiguration.m_topic = "cmd_vel";
```

Comment:
remove assignments that repeat the default value

Reply:
Defaults are :
```
        AZStd::string m_topic = "o3de_robot_control";
                bool m_broadcastBusMode = true;
```  
Adjust defaults then?

Reply:
just remove assignments here (all 3) due to vehicle dynamics PR resolving these